.babelrc

```diff
@@ -1,6 +1,11 @@
 {
   "presets": [
-    ["env", { "loose": true, "targets": { "node": "4.2" } }]
+    ["@babel/preset-env", {
```

Comment:
You can leave off the `preset-` if you want.

Also looks like there are a bunch of things in the README that will need updating.

Reply:
ok updated